### ISSUE TITLE
Changing table saving data no errors but actually save is not done

### ISSUE THREAD

**Issue by goxr3plus** [dbeaver/dbeaver#5975]
There is a strange behaviour with DBeaver , i have noticed since edition 5.

Let's say i am connected to a database and i am trying to change the table data . 

I do change the data and it the table ( see blue box , and i press save) :

![dbeaver_2019-05-22_13-04-26](https://user-images.githubusercontent.com/20374208/58166342-4bc85380-7c92-11e9-8dea-4e611b31a20a.png)

Then i disconnect and reconnect to the database :

![dbeaver_2019-05-22_13-04-55](https://user-images.githubusercontent.com/20374208/58166359-54b92500-7c92-11e9-9de7-0757725de476.png)

![dbeaver_2019-05-22_13-05-02](https://user-images.githubusercontent.com/20374208/58166367-584cac00-7c92-11e9-87bc-c3df3a10d5f4.png)

**And the value is still what it was before i saved , like nothing never changed even if i saved and no errors :**

![2019-05-22_13-04-46](https://user-images.githubusercontent.com/20374208/58166398-68fd2200-7c92-11e9-9d7b-56da229ba670.png)

No errors , nothing.

The strange is that i tried to do the same with **MySqlWorkbench** it reports error . Why?

**Comment by p91paul** [comment]
did you commit after save? commit button is at the top...

**Comment by goxr3plus** [comment]
Let me try again, isn't save enough?

**Comment by p91paul** [comment]
unless auto commit is on, it's not enough. save writes to the database, but until you commit your data is not visible to other sessions. This is the standard way transactions work!

**Comment by goxr3plus** [comment]
@p91paul  Hello Paul you are right , i will close the issue , how to turn the auto-commit on :) ?

**Comment by p91paul** [comment]
edit connection -> initialization (if I remember correctly)

**Comment by goxr3plus** [comment]
@p91paul :)
